pages/mesosphere/dcos/1.14/gui/networking/index.md

```diff
@@ -8,8 +8,80 @@ render: mustache
 model: /mesosphere/dcos/1.14/data.yml
 ---
 
-The Networking page provides a comprehensive view of the health of your virtual IP addresses (VIPs).
+The Networking screen provides a comprehensive view of the health of your virtual IP addresses (VIPs).
 
-![Tweeter scaled](/mesosphere/dcos/1.14/img/GUI-Networking-Networks_View-1_12.png)
+
+![Tweeter scaled](/1.14/img/GUI-Networking-Main.png)
 
 Figure 1 - Networking tab
+
+There are two sub-pages for the Networking interface: **Networks** and **Service Addresses**.
+
+# Networks tab
+
+The **Networks** tab lists the networks currently deployed on your cluster. This is the main view, by default. 
+
+If you click on the name of a network, you can see the Networks detail page, with two tabs: **Tasks** and **Details**.
+
+![Networks Detail](/1.14/img/GUI-Networking-Networks-Detail.png)
+
+Figure 2 - Networks detail
+
+## Tasks
+
+The **Tasks** tab lists the following information:
+
+| Name | Description |
+|---------|--------------|
+| Task |   |
+| Container IP |        |
+| Port Mappings |        |
+
+## Details
+
+The **Details** tab lists the following information:
+
+| Name | Description |
+|---------|--------------|
+| Name |   |
+| IP Subnet |        |
+
+# Service Addresses tab
+
+The Service Addresses tab displays a summary of the statistics for each network service address:
+
+![Service Addresses tab](/1.14/img/GUI-Networking-Service-Addresses-Main.png)
+
+Figure 3 - Service Addresses tab
+
+| Name | Description |
+|---------|--------------|
+| Name |   |
+| Successes |        |
+| Failures | |
+| Failure % | |
+| P99 Latency |   |
```

Comment:
P99 latency means that 99% of the requests should be faster than given latency. In other words only 1% of the requests are allowed to be slower.